docs/UserGuide.md

```diff
@@ -1,33 +1,94 @@
-# User Guide
+# ra.VI User Guide
 
 ## Introduction
 
-{Give a product intro}
+Repository Assistant with a Versatile Interface (ra.VI) is a desktop app for managing tasks, deadlines, and notes, optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, ra.VI can manage your tasks faster than traditional GUI apps.
 
 ## Quick Start
 
 {Give steps to get started quickly}
 
 1. Ensure that you have Java 11 or above installed.
-1. Down the latest version of `Duke` from [here](http://link.to/duke).
+2. Down the latest version of `ra.VI` from [here](http://link.to/duke).
+3. Copy the file to the folder you want to use as the home folder for your scheduler. Double-click the file to start the app. 
+4. Type the command in the command box and press Enter to execute it. e.g. typing help and pressing Enter will open the help window.  
+Some example commands you can try:
+* `add -t`: Add a task
+* `list -t`: List all tasks
+* `exit`: Exit the program read chapter 1`
+5. Refer to the Features below for details of each command
 
 ## Features 
 
 {Give detailed description of each feature}
+### Viewing help: `help`
+Shows a message with the list of available commands and functions.
 
-### Adding a todo: `todo`
-Adds a new item to the list of todo items.
+### Adding an item: `add <opt><args>`
+#### Adding a task: `add -t [-by]`
+Add a task to the scheduler.  
+Format: `add -t <task_name> [-by <deadline>]`
 
-Format: `todo n/TODO_NAME d/DEADLINE`
+Example of usage: 
+* `add -t read chapter 1`
+* `add -t read chapter 1 -by 12 Sep`
 
-* The `DEADLINE` can be in a natural language format.
-* The `TODO_NAME` cannot contain punctuation.  
+#### Adding a module: `add -m`
+Add a module to the scheduler.  
+Format: `add -m <module_code>`
 
 Example of usage: 
+* `add -m CS1231`
+
+### Editing an item: `edit <opt><args>`
+#### Editing a task description: `edit -t <task_index> <args>`
+Edit a task description in the scheduler.  
+Format: `edit -t <task_index> <args>`
+
+Example of usage: 
+* `edit -t 3 revise for CS2113`
+
+#### Editing a module: `edit -m <module_code> <new_module_code>`
+Edit a module code in the scheduler.  
+Format: `edit -m <module_code> <new_module_code>`
 
-`todo n/Write the rest of the User Guide d/next week`
+Example of usage: 
+* `edit -m “CS2113” “CS2113T”`
+
+### Deleting an item: `del <opt><args>`
+#### Deleting a task: `del -t <task_index>`
+Remove a task from the scheduler.  
```

Comment:
Should add an explanation here that the `task_index` can be found from `list -t`

Reply:
Agreed, because the task index can only be found through list